Library/Formula/fpc.rb

```diff
@@ -28,11 +28,18 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to facilitate search paths
+    fpcversion = `./compiler/ppc1 -iW`.chomp
+    fpccpu     = `./compiler/ppc1 -iTP`.chomp
+    fpcos      = `./compiler/ppc1 -iTO`.chomp
+    system "./utils/fpcmkcfg/bin/#{fpccpu}-#{fpcos}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{fpcversion}", "-o", "#{prefix}/etc/fpc.cfg"
```

Comment:
I think these 4 lines can be squashed into `system #{bin}/fpcmkcfg", ...` since `fpcmkcfg` should be already built and installed in `bin` at this point.